archetypes/anti-ventrue-grinder.html

```diff
@@ -1,158 +0,0 @@
-<!DOCTYPE html>
-<html lang="en-GB">
-<head>
-	<meta charset="UTF-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
-    <meta property="og:image" content="https://codex-of-the-damned.org/codex.jpg"/>
-    <meta name="description" content="Vampire: the Eternal Struggle (VTES) archetype: Anti Ventrue Grinder">
```

Comment:
We are losing this information in the PR i made earlier. Maybe use the nav to implement it in the archetypes/layout ?

Reply:
Yes, I can look into it if you want while you progress on other stuff - I actually would love to get a custom `og:image` header for the card search page

Reply:
It'd be wonderful if you look into it will i retemplate the rest of the site